src/common/entity.ts

```diff
@@ -158,6 +158,10 @@ export class Enemy extends Entity {
     throw new Error('Method not implemented.');
   }
 
+  public move(): void {
+    this._position.add(Vec2(0, 1));
+  }
+
```

Comment:
den här ska nog inte vara kvar i slutändan, men den borde använda `this._velocity` istället för `Vec2(0, 1)` i alla fall.